pkg/controller/datavolume/import-controller.go

```diff
@@ -274,14 +269,16 @@ func (r *ImportReconciler) updateStatusPhase(pvc *corev1.PersistentVolumeClaim,
 		event.message = fmt.Sprintf(MessageImportFailed, pvc.Name)
 	case string(corev1.PodSucceeded):
 		if _, ok := pvc.Annotations[cc.AnnCurrentCheckpoint]; ok {
-			if err := r.updatesMultistageImportSucceeded(pvc, dataVolumeCopy); err != nil {
-				return err
+			if !importPopulation {
```

Comment:
Can you explain why you are not updating the status in case of populators? cause I do see you update the volumesource according to the datavolume, so how come the datavolume is not updated according to it?

Reply:
This function updates the PVC annotations according to the state of the multi-stage import once the import pod is succeeded. The import populator already does that, so we don't need to call the same function here. The datavolume is updated as before.